File: week1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // prompt user for height of pyramid
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height <= 0);

    // loop to print pyramid
    for (int i = 0; i < height; i++)
    {
        // prints left half of pyramid
        for (int j = 0; j < height - i - 1; j++)
        {
            printf(" ");
        }
        for (int k = 0; k < i + 1; k++)
        {
            printf("#");
        }

        // prints space in mid of pyramid
        printf("  ");

        // prints right half of pyramid
        for (int m = 0; m < i + 1; m++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: week6/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height > 0 and height <= 8:
        break

for i in range(height):
    # left half of pyramid
    for x in range(height - i - 1):
        print(" ", end="")
    for y in range(i + 1):
        print("#", end="")
    # space between left & right half
    print("  ", end="")
    # right half of pyramid
    for z in range(i + 1):
        print("#", end="")
    print()
